src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10053,6 +10092,32 @@ private bool ScanParenthesizedImplicitlyTypedLambda(Precedence precedence)
             }
 
             return false;
+
+            bool isParenVarCommaSyntax()
```

Comment:
make static if possible.

Reply:
The function uses 'this'. Are you asking to convert so that everything captured is instead passed as a parameter?

Reply:
Tried, doesn't allow me to PeekToken or test if it's in query; want me to extract them as params or leave as non static?

Reply:
nope.  this is acceptable :)    it was a more a suggestion of "if you can, please do this".  if it's onerous, you def don't have to :)

Reply:
No worries, just thought it best to check first!